src/screens/Events/Admin/Statistics/EditEventStatsForm.jsx

```diff
@@ -33,16 +34,32 @@ const Styled = {
   `,
 };
 
+const timeValidator = object().shape({
```

Comment:
Can this be moved to the eventHelpers file?

Reply:
done